dataprocspawner/spawner.py

```diff
@@ -149,6 +158,115 @@ def _validate_proto_field(data, field_descriptor):
   # Other types we don't attempt to validate
   return True, warnings
 
+
+class DataprocHubServer(Server):
```

Comment:
Let's put this class in a separate file, since this spawner.py file is getting pretty long.

Reply:
Fixed in 43f039ff95789b660d341d9741bb13019b470a06